kaldb/src/test/java/com/slack/kaldb/chunkManager/ChunkCleanerServiceTest.java

```diff
@@ -197,7 +201,7 @@ private void testBasicSnapshotMetadata(Instant creationTime) {
     assertThat(snapshotNodes.get(0).partitionId).isEqualTo(TEST_KAFKA_PARTITION_ID);
     assertThat(snapshotNodes.get(0).snapshotId).startsWith(SnapshotMetadata.LIVE_SNAPSHOT_PATH);
     assertThat(snapshotNodes.get(0).startTimeEpochMs)
-        .isCloseTo(creationTime.toEpochMilli(), Offset.offset(1000L));
+        .isCloseTo(creationTime.toEpochMilli(), Offset.offset(2500L));
```

Comment:
this will make it less flaky?

Reply:
It should - I was having some intermittent time issues as well running this locally, as 1s appeared to be a bit too strict.